libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -217,6 +217,7 @@ These are the fields available within config templating. The `kubernetes.*` fiel
 
 [float]
 ====== Pod specific:
+  * container.id
```

Comment:
I think we also expose `container.image.name` field too and maybe others. Shall we check about them and have an updated listing here?

Reply:
I am not sure what are the other fields or where to find them. I have added `container.image.name` as you requested.

Reply:
It can only be found in the code, I think it is https://github.com/elastic/beats/blob/c1d4f28ff30e28429efed51c817cd02d46633aac/libbeat/autodiscover/providers/kubernetes/pod.go#L326-L332 but verifying this by running metricbeat in debug mode and checking the logged autodiscovery events would provide the best confidence :).